app/services/support_interface/work_history_break_export.rb

```diff
@@ -0,0 +1,221 @@
+            'Total time volunteering during unexplained breaks (months)' => total_time_volunteering_during_breaks(unexplained_breaks, volunteering_experiences),
+            'Number of unexplained breaks' => unexplained_breaks.length,
+            'Number of unexplained breaks in last 5 years' => breaks_in_last_five_years(unexplained_breaks, application_form),
+            'Number of unexplained breaks that coincide with studying for a degree' => unexplained_breaks_that_coincide_with_degrees(application_form, unexplained_breaks),
+            'Number of unexplained breaks that coincide with a volunteering experience' => breaks_that_coincide_with_volunteering_experiences(unexplained_breaks, volunteering_experiences),
+            'Number of unexplained breaks that were over 50% volunteering' => breaks_with_over_fifty_percent_volunteering(unexplained_breaks, volunteering_experiences),
+          })
+        end
+
+        output
+      end
+
+      # The DataExport class creates the header row for us so we need to ensure
+      # we sort by longest hash length to ensure all headers appear
+      data_for_export.compact.sort_by(&:length).reverse
+    end
+
+  private
+
+    def start_of_working_life(application_form)
+      application_form.date_of_birth.beginning_of_month + 18.years
+    end
+
+    def total_time_in_employment(application_form)
+      time_in_seconds = application_form.application_work_experiences.map { |experience| experience.end_date - experience.start_date }
+      convert_seconds_to_months(time_in_seconds.sum)
+    end
+
+    def total_time_of_explained_breaks(explained_breaks)
+      time_in_seconds = explained_breaks.map { |work_history_break| work_history_break.end_date - work_history_break.start_date }
+      convert_seconds_to_months(time_in_seconds.sum)
+    end
+
+    def total_time_of_unexplained_breaks(unexplained_breaks)
+      unexplained_breaks.sum(&:length)
+    end
+
+    def total_time_volunteering_during_breaks(breaks, volunteering_experiences)
+      total_time = 0
+      breaks.each do |b|
+        volunteering_experiences_during_break(b, volunteering_experiences).each do |v|
+          total_time += volunteering_time_during_break(b, v)
+        end
+      end
+      convert_seconds_to_months(total_time)
+    end
+
+    def volunteering_time_during_break(work_break, volunteering_experience)
+      volunteering_end_date = volunteering_experience.end_date ||= Time.zone.now
+
+      start_date = work_break.start_date < volunteering_experience.start_date ? volunteering_experience.start_date : work_break.start_date
+      end_date = work_break.end_date > volunteering_end_date ? volunteering_end_date : work_break.end_date
+      end_date - start_date
+    end
+
+    def breaks_in_last_five_years(breaks, application_form)
+      breaks.count { |b| b.start_date > (submitted_at(application_form) - 5.years).to_date }
```

Comment:
Depending on what you and Despo agree regarding short vars, then you may need to update here for consistency